image/oci/manifest.py

```diff
@@ -354,11 +354,11 @@ def _manifest_image_layers(self, content_retriever):
             return
 
         # Make sure we aren't missing any history entries if it was specified.
-        if len(history) < len(self.filesystem_layers):
-            raise MalformedOCIManifest(
-                "Found less history (%s) than layer blobs (%s)"
-                % (len(history), len(self.filesystem_layers))
-            )
+        # if len(history) < len(self.filesystem_layers):
+        #     raise MalformedOCIManifest(
+        #         "Found less history (%s) than layer blobs (%s)"
+        #         % (len(history), len(self.filesystem_layers))
+        #     )
```

Comment:
Are removing these checks required for the feature to work? Can we add these back in for now?

Reply:
I added these back. I think the correct way if for the images to have an empty descriptor instead